app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -80,9 +74,7 @@ fun WallpaperSettings(
     loadWallpaperResource: (Wallpaper) -> Bitmap?,
     selectedWallpaper: Wallpaper,
     onSelectWallpaper: (Wallpaper) -> Unit,
-    onViewWallpaper: () -> Unit,
-    tapLogoSwitchChecked: Boolean,
-    onTapLogoSwitchCheckedChange: (Boolean) -> Unit
+    onViewWallpaper: () -> Unit
```

Comment:
nit: would you add a trailing comma here? I am not sure if we have this referenced anywhere, but we have been trying to include them wherever possible. It tends to make diffs in future patches a little easier to read, as it'll reduce the number of lines highlighted.